pkg/config/config_template.yaml

```diff
@@ -3469,12 +3469,19 @@ api_key:
       #
       # mode: distributions
 
+      ## Deprecated - use `send_aggregation_metrics` instead
       ## @param send_count_sum_metrics - boolean - optional - default: false
       ## @env DD_OTLP_CONFIG_METRICS_HISTOGRAMS_SEND_COUNT_SUM_METRICS - boolean - optional - default: false
-      ## Whether to report sum and count as separate histogram metrics.
+      ## Whether to report sum, count, min and max as separate histogram metrics.
       #
       # send_count_sum_metrics: false
 
+      ## @param send_aggregation_metrics - boolean - optional - default: false
+      ## @env DD_OTLP_CONFIG_METRICS_HISTOGRAMS_SEND_AGGREGATION_METRICS - boolean - optional - default: false
+      ## Whether to report sum, count, min and max as separate histogram metrics.
+      #
+      # send_aggregation_metrics: false
+
```

Comment:
I think we should explain which on will take precedence when both are set.

Reply:
> I think so, having to deploy new configuration and restart agent to detect errors is not a great workflow for customers.

I am used to the OpenTelemetry Collector way of doing things, where the approach is 'fail fast if anything is wrong/ambiguous'. I think it works great there and this approach helps us prevent hard-to-detect faulty deployments, but I don't want to be inconsistent with how the rest of the Agent works just because I am used to that.

> In the past, for similar cases, we gave precedence on the old value and printed a warning.

I will do that then :+1:

Reply:
Fixed in e5a6b34e8d418048a634cbeb8d2988f45a5c5940